src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
```

Comment:
I think this blank line can be removed? (So that it follows the formatting of the rest of your code, where the class's attributes are directly under the class call)